### PR TITLE
[Motion lwg 1] P2910R0 C++ Standard Library Issues

### LINKED ISSUE #6287: [2023-06 LWG Motion 1] Tentatively Ready issues in P2910R0 C++ Standard Library Issues to be moved in Varna, Jun. 2023

### PR DESCRIPTION
**Pull request by burblebee**
Fixes #6287.

Note: LWG3887 skipped - already fixed.

### CODE REVIEW

File: source/containers.tex [hunk 1]
```diff
@@ -15166,6 +15171,10 @@
 
 \indexlibraryctor{flat_map}%
 \begin{itemdecl}
+template<class Allocator>
+  flat_map(const flat_map&, const Allocator& a);
+template<class Allocator>
+  flat_map(flat_map&&, const Allocator& a);
 template<class Allocator>
   flat_map(const key_compare& comp, const Allocator& a);
 template<class Allocator>
```
Comment (jwakely) [inline source/containers.tex:15180]: This is missing the drive-by fix to the Constraints paragraph.

**Review by jwakely — APPROVED** [review]
Looks good!

**Review by JohelEGP — APPROVED** [review]
I'm using <https://isocpp.org/files/papers/P2910R0.html>.

> Note: [LWG3887](https://wg21.link/lwg3887) skipped - already fixed.

For the record, the commit that fixed it is <https://github.com/cplusplus/draft/commit/6e7fd98b684a92405934de981abf01414a67ddbd>.
